### PR TITLE
IA-2369 add publishNativeOne

### PR DESCRIPTION
**Pull request by Qi77Qi**
**PR checklist**
- [ ] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [ ] Bump the version in `project/Settings.scala` `createVersion()`
- [ ] Update `CHANGELOG.md` for those libraries
- [ ] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [ ] Replace the appropriate version hashes in `README.md` and the `CHANGELOG.md` for any libs you changed with `TRAVIS-REPLACE-ME` to auto-update the version string
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green (CI _and_ coverage tests)
- [ ] Squash commits and **merge to develop**
- [ ] Delete branch after merge

### CODE REVIEW

File: google2/CHANGELOG.md [hunk 1]
```diff
@@ -2,25 +2,21 @@
 
 This file documents changes to the `workbench-google2` library, including notes on how to upgrade to new versions.
 
-## 0.18
```
Comment (Qi77Qi) [inline google2/CHANGELOG.md:5]: version wasn't bumped properly in previous PR..hence, just moving this to 0.17...

File: README.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ Workbench utility libraries, built for Scala 2.12 and 2.13. You can find the ful
 
 Contains utility functions for talking to Google APIs via com.google.cloud client library (more recent) via gRPC. 
 
-Latest SBT dependency: `"org.broadinstitute.dsde.workbench" %% "workbench-google2" % "0.18-626dbbc"`
```
Comment (kyuksel) [inline README.md:75]: `develop` is at version `0.18`. May want to rebase.

File: google2/src/main/scala/org/broadinstitute/dsde/workbench/google2/GooglePublisherInterpreter.scala [hunk 3]
```diff
@@ -30,23 +30,28 @@ private[google2] class GooglePublisherInterpreter[F[_]: Async: Timer: Structured
...
+          publisher.publish(message),
+          callBack(callback),
+          MoreExecutors.directExecutor()
+        )
+      }
+      .void,
+    Option(message.getAttributesMap.get("traceId")).map(s => TraceId(s)),
+    s"Publishing ${message}"
+  )
+
```
Comment (kyuksel) [inline google2/src/main/scala/org/broadinstitute/dsde/workbench/google2/GooglePublisherInterpreter.scala:54]: Could you explain the significance of `Native` in some of these method names?

**Review by kyuksel — CHANGES_REQUESTED** [review]
Looks good overall. I think it needs rebasing, though.

File: google2/src/main/scala/org/broadinstitute/dsde/workbench/google2/GooglePublisherInterpreter.scala (hunk 3 above)
Comment (Qi77Qi) [reply google2/src/main/scala/org/broadinstitute/dsde/workbench/google2/GooglePublisherInterpreter.scala:54]: `Native` just meant the function takes a `PubsubMessage` object directly which comes from google's library

File: README.md (hunk 2 above)
Comment (Qi77Qi) [reply README.md:75]: this was intentional at the time becuz I thought last PR had a mistake not bumping the version. But it seems like it did, so I reverted this change

**Review by kyuksel — APPROVED** [review]
:shipit:

**Pull request merged**